lib/merge_hris_client/api/employees_api.rb

```diff
@@ -228,6 +236,10 @@ def employees_list_with_http_info(x_account_token, opts = {})
       if @api_client.config.client_side_validation && x_account_token.nil?
         fail ArgumentError, "Missing the required parameter 'x_account_token' when calling EmployeesApi.employees_list"
       end
+      allowable_values = ["ACTIVE", "INACTIVE", "PENDING"]
+      if @api_client.config.client_side_validation && opts[:'employment_status'] && !allowable_values.include?(opts[:'employment_status'])
```

Comment:
Probably dont want this? seems like it'll break if a value doesnt map to our enum

Reply:
auto-generated, similarly exists in various other api files as well